src/goImport.ts

```diff
@@ -20,10 +20,18 @@ export async function listPackages(excludeImportedPkgs: boolean = false): Promis
 		? await getImports(vscode.window.activeTextEditor.document)
 		: [];
 	const pkgMap = await getImportablePackages(vscode.window.activeTextEditor.document.fileName, true);
-
+	const stdLib = await getStandardLibraryPackages();
 	return Array.from(pkgMap.keys())
 		.filter(pkg => !importedPkgs.some(imported => imported === pkg))
-		.sort();
+		.sort((a,b)=> {
+			if (stdLib.get(a)){
+				return -1
+			}
+			if (stdLib.get(b)){
+				return 1
+			}
```

Comment:
This sorting will only ensure that the standard libraries appear before the others.
But among the standard libraries themselves, we lose the sorting
Similarly, we also lose the sorting among the non standard libraries